chef_master/source/custom_resources_notes.rst

```diff
@@ -181,9 +181,9 @@ If you do need to write code which mutates the system through pure-Ruby then you
      end
    end
 
-The ``converge_by`` block gets why-run correct and will just touch "/tmp/foo" instead of actually doing it. The ``converge_by`` block is also responsible for setting ``update_by_last_action``.
+The ``converge_by`` block gets why-run correct and will print to ``Chef::Log.info`` "touch /tmp/foo" and not run ``::FileUtils.touch "/tmp/foo"``. The ``converge_by`` block is also responsible for setting ``update_by_last_action``.
```

Comment:
This is awkward reading.

The prior text was incorrect and mangled somehow.

When the ``converge_by`` block is run in why-run mode, it will only log ``touch "/tmp/foo"`` and will not run the code inside the block.

I really dislike throwing the `Chef::Log.info` and `FileUtils.touch` details at the reader.  It is also likely better to be clear that the code inside the block is not executed -- there is no magic other than the block simply is not run.  Also "gets why-run correct" is awkward (i think I wrote that one initially, it sounds like me) and should really be changed.